packages/server-wallet/src/models/__test__/objectives.test.ts

```diff
@@ -44,6 +44,29 @@ describe('Objective > insert', () => {
       {objectiveId: `OpenChannel-${c.channelId}`, channelId: c.channelId},
     ]);
   });
+
+  it('inserts an objective with a createdAt timestamp', async () => {
+    await Channel.query(knex).withGraphFetched('signingWallet').insert(c);
+
+    const before = Date.now() - 1000; // scroll back 1000 ms to allow for finite precision / rounding
+    const {createdAt} = await ObjectiveModel.insert({...objective, status: 'pending'}, knex);
+    const after = Date.now() + 1000; // scroll forward 1000 ms to allow for finite precision / rounding
+
+    expect(Date.parse(createdAt) > before).toBe(true);
+    expect(Date.parse(createdAt) < after).toBe(true);
+  });
+
+  it('updates the timestamp on an objective when it succeeds', async () => {
+    await Channel.query(knex).withGraphFetched('signingWallet').insert(c);
+    const {objectiveId} = await ObjectiveModel.insert({...objective, status: 'pending'}, knex);
+
+    const before = Date.now() - 1000; // scroll back 1000 ms to allow for finite precision / rounding
+    const {updatedAt} = await ObjectiveModel.succeed(objectiveId, knex);
```

Comment:
I'm confused here, since I didn't think `updatedAt` was a property on objectives?

Reply:
This was a mistake I made, but I think I fixed it before you posted your comment (maybe you started your review a while ago?)

Reply:
I often review commits atomically, so I came across `updatedAt` and it hadn't been defined yet in the git history.